app/src/main/res/values/styles.xml

```diff
@@ -468,6 +468,28 @@
     <item name="android:textAlignment">viewStart</item>
   </style>
 
+  <style name="AddMultiProfileHeading" parent="TextView.Common1">
+    <item name="android:textAlignment">viewStart</item>
+    <item name="fontFamily">sans-serif-medium</item>
+    <item name="android:textColor">@color/color_def_white</item>
+    <item name="android:textSize">14sp</item>
+  </style>
+
+  <style name="AddMultiProfileBody" parent="TextView.Common1">
+    <item name="android:textAlignment">viewStart</item>
+    <item name="fontFamily">sans-serif</item>
+    <item name="android:textColor">@color/color_def_white</item>
+    <item name="android:textSize">12sp</item>
+  </style>
+
+  <style name="ShapeAbleImageView" parent="ShapeAppearanceOverlay.RoundedShape">
+    <item name="strokeWidth">1dp</item>
+    <item name="strokeColor">@color/component_color_profile_chooser_activity_avatar_border_color</item>
```

Comment:
Please use `component_color_shared_divider_color` instead of `component_color_profile_chooser_activity_avatar_border_color`

Reply:
> Please use `component_color_shared_divider_color` instead of `component_color_profile_chooser_activity_avatar_border_color`

Sure, that's what's used here already.

Reply:
No, in strokeColor now it is `@color/component_color_profile_chooser_activity_avatar_border_color` but use `@color/component_color_shared_divider_color`